source/tutorials/write-update-documents.txt

```diff
@@ -0,0 +1,291 @@
+``updateMany()`` methods.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. 
+
+.. TODO:
+.. Once get started ticket merged in, link the get started tutorial below
+
+.. To learn how to create a
+.. free MongoDB Atlas cluster and load the sample datasets, see the
+.. :ref:`<java-rs-getting-started>` tutorial.
+
+.. TODO: When write insert methods merged in, add subscriber methods section includes within subscriber methods section
+
+.. Subscriber Methods
+.. ~~~~~~~~~~~~~~~~~~~
+
+Update Operations
+-----------------
+
+You can perform update operations in MongoDB by using the following methods:
+
+- ``updateOne()``, which updates *the first document* that matches the search criteria
+- ``updateMany()``, which updates *all documents* that match the search criteria
+
+Each update method requires the following parameters:
+
+- A query filter document, which determines which documents to update. For more
+  information about using query filters, see :ref:`Filters <java-rs-update-filters>`.
+
+- An update document, which specifies the update operator (the kind of update to
+  perform) and the fields and values that should change. For more information
+  about update documents, see :ref:`Update Operators <java-rs-update-operators>`.
+
+.. _java-rs-update-filters:
+
+Filters
+~~~~~~~
+
+Each update method requires a **query filter**, which specifies the
+search criteria that determine which documents to select for updates. To facilitate the creation of filter objects, the
+driver provides the ``Filters``class that provides filter condition helper
+methods. 
+
+To view a list of ``Filters`` helpers, see the `Filters
+API documentation <{+api+}/mongodb-driver-core/com/mongodb/client/model/Filters.html>`__.
+For more information about query filters, see the 
+:manual:`Query Filter Documents section </core/document/#query-filter-documents>` in
+the MongoDB Server manual. 
+
+.. _java-rs-update-operators:
+
+Update Operators
+~~~~~~~~~~~~~~~~
+
+To change a field in a document, MongoDB provides update operators.
+To specify the modification to perform using the update operators,
+create an update document. To learn more about update operators, see
+:manual:`Update Operators </reference/operator/update/>` in the Server manual.
+
+To facilitate the creation of update documents, the driver
+provides the ``Updates`` helper class.
+
+.. important::
+
+   The ``id`` field is immutable, so you cannot change the value of the
+   ``id`` field in a document.
+
+Update One Document
+-------------------
+
+To update a single document from a MongoDB collection, call the ``updateOne()``
+method and pass in your query filter as the first argument. Then, pass in your
+update operators as the second argument to the ``updateOne()`` method. After you call the
+``updateOne()`` method, chain the ``subscribe()`` method and pass in a subscriber.
```

Comment:
S: use "pass" instead of "pass in"
```suggestion
method and pass your query filter as the first argument. Then, pass your
update operators as the second argument to the ``updateOne()`` method. After you call the
``updateOne()`` method, chain the ``subscribe()`` method and pass a subscriber as an argument.
```

Reply:
Applies to all